src/main/app/src/app/app.component.ts

```diff
@@ -31,6 +33,8 @@ export class AppComponent implements OnInit, AfterViewInit {
         this.translate.setDefaultLang('nl');
         const browserLanguage = this.translate.getBrowserLang();
         this.translate.use(browserLanguage.match(/nl|en/) ? browserLanguage : 'nl');
+        SessionStorageUtil.removeItem(IdentityService.LOGGED_IN_USER_KEY);
+        this.identityService.readLoggedInUser();
```

Comment:
Er zijn geen subscribers, zou de readLoggedInUser gewoon weglaten. de eerste die deze call aanroept laad hem wel in de sessionstorage